impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterFactory.java

```diff
@@ -74,6 +73,12 @@ public static KafkaCluster create(KafkaClusterConfig clusterConfig) {
             builder.kraftMode(kraftMode);
         }
 
+        if (KafkaClusterExecutionMode.CONTAINER == clusterMode && kraftMode && clusterConfig.getBrokersNum() < clusterConfig.getKraftControllers()) {
+            throw new IllegalStateException(
+                    "Due to https://github.com/ozangunalp/kafka-native/issues/88 we can't support controller only nodes so we need to fail fast. This cluster has "
```

Comment:
maybe this log should mention the execution mode, the other one doesn't need to since the classname contains TestContainers